website/docs/components/form/segmented-group/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,116 @@
+## Usage
+
+A Segmented Group is an agnostic container that wraps two or more `Segments`, making the relevant use cases broad.
+
+!!! Info
+
+The way in which these components are published and assembled is fundamentally different between the Figma and Ember components.
+
+- In Figma, we publish multiple primitive `Segment` components (`Segmented Button`, `Segmented Toggle`, and `Segmented Input`) that are intended to be assembled in an auto layout container. A pre-assembled `Base` component is also made available for more simple instances.
+- The Ember component is a container that enforces consistent styling and grouping of nested Helios form components. It does not manage any logic or functionality out of the box, these are left up to the consumer to implement based on the nested components.
+!!!
+
+### When to use
+
+- When filtering complex data, searching within a set of filters, and in creating complex filter patterns.
+- When collecting information in a way that benefits from a "connected" visual representation, e.g., a key/value pair.
+- When an action performs a generative function rather than a submit function, e.g., generating API credentials.
+- As a complex data input within a form.
+
+### When not to use
+
+- To submit a form or set of inputs. A Segmented Group can be used within a form to collect complex data, but should not submit that data via a form method. Use the guidelines in [Form patterns](/patterns/form-patterns) instead.
```

Comment:
🤔 hmm, not sure about this – why would we not want them to submit the data using a standard form? happy to catch-up on this

Reply:
ah, I see! yes, that sounds like valuable guidance – may not require rewording, it was probably just me reading through the lines 😅